daprdocs/content/en/reference/cli/dapr-run.md

```diff
@@ -42,6 +42,8 @@ dapr run [flags] [command]
 | `--enable-api-logging`         |                      | `false`                                                                            | Enable the logging of all API calls from application to Dapr      |
 | `--metrics-port`               | `DAPR_METRICS_PORT`  | `9090`                                                                             | The port that Dapr sends its metrics information to                                                  |
 | `--profile-port`               |                      | `7777`                                                                             | The port for the profile server to listen on                                                         |
+| `--placement-host-address`      |                      | Linux/Mac: `$HOME/.dapr/components` <br/>Windows: `%USERPROFILE%\.dapr\components` | Run in any containers within your Docker network. Uses `<hostname>` or `<hostname>:<port>`. If the port is omitted, it will default to: <ul><li>Linux/MacOS: `50005`</li><li>Windows: `6050`</li></ul> |
+| `--scheduler-host-address`      |                      | Linux/Mac: `$HOME/.dapr/components` <br/>Windows: `%USERPROFILE%\.dapr\components` | Run in any containers within your Docker network. Uses `<hostname>` or `<hostname>:<port>`. If the port is omitted, it will default to: <ul><li>Linux/MacOS: `50005`</li><li>Windows: `6050`</li></ul> |
```

Comment:
```suggestion
| `--scheduler-host-address`      |                      | Linux/Mac: `$HOME/.dapr/components` <br/>Windows: `%USERPROFILE%\.dapr\components` | Run in any containers within your Docker network. Uses `<hostname>` or `<hostname>:<port>`. If the port is omitted, it will default to: <ul><li>Linux/MacOS: `50006`</li><li>Windows: `6060`</li></ul> |
```